d-sgov/d-sgov-vfr-1.8.0-diagram.ttl

```diff
@@ -149,6 +149,12 @@ d-sgov-vfr-1.8.0:bonitovaný-díl-parcely-class-diagram
         <http://www.w3.org/2004/02/skos/core#prefLabel>
                 "bonitovaný díl parcely class diagram"@cs .
 
+d-sgov-vfr-1.8.0:propojení-ofn-a-vfr
+        <http://purl.org/dc/terms/identifier>
+                "4595ed31-07df-4069-9338-2142df1e4d15" ;
```

Comment:
Kde se vzal tento identifikátor?

Reply:
To bude nějakej hash z modelia. Každopádně koukám, že to je prázdný artefakt, který deletuji.